docs/source/55_api.md

```diff
@@ -339,6 +339,52 @@ val bcastFeatures = sc.loadFeatures("my/features.adam").broadcast()
 val readsByFeature = reads.broadcastRegionJoinAgainst(bcastFeatures)
 ```
 
+#### Examples of real-world analyses possible with the RegionJoin API
+###### Filter Genotypes by Features
+
+```scala
+// Inner join will filter out genotypic data not represented in the feature dataset
+val genotypes = sc.loadGenotypes(“my/genotypes.adam”)
+val features = sc.loadFeatures(“my/features.adam”)
+
+// We can use ShuffleRegionJoin…
+val filteredGenotypes_shuffle = genotypes.shuffleRegionJoin(features)
+
+// …or BroadcastRegionJoin
+val filteredGenotypes_bcast = genotypes.broadcastRegionJoin(features)
+```
+
+###### Group overlapping variant data by the gene they overlap
+```scala
+// Inner join with a group by on the features
+val features = sc.loadFeatures(“my/features.adam”)
+val variants = sc.loadVariants(“my/variants.adam")
+
+// As a ShuffleRegionJoin, it can be implemented as follows:
+val variantsByFeature_shuffle = features.shuffleRegionJoinAndGroupByLeft(variants)
+
+// As a BroadcastRegionJoin, it can be implemented as follows:
+val variantsByFeature_bcast = variants.broadcastRegionJoinAndGroupByRight(features)
```

Comment:
1. Please highlight that the sides of the join have swapped between the two queries. This didn't stick out the first time I read through.
2. Please add text discussing the performance implications of rewriting this as a broadcast join.